lib/screen/new_onboarding_page.dart

```diff
@@ -261,28 +268,22 @@ class _NewOnboardingPageState extends State<NewOnboardingPage> {
           desc: didUserBuy
               ? 'thank_for_being_pro_desc'.tr()
               : 'you_received_premium_desc'.tr(),
-          subDescFixedSized: false,
-          subDesc: Padding(
-            padding: const EdgeInsets.only(top: 30, right: 15, left: 15),
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.spaceBetween,
-              children: [
-                MembershipCard(
-                  type: MembershipCardType.premium,
-                  price: _getPremiumPrice(subscriptionDetails),
-                  isProcessing: false,
-                  isEnable: false,
-                ),
-                Padding(
-                  padding: const EdgeInsets.only(bottom: 40),
-                  child: PrimaryButton(
-                    text: 'continue'.tr(),
-                    onTap: () {
-                      _goToHomePage(context);
-                    },
-                  ),
-                ),
-              ],
+          subscriptionInfo: MembershipCard(
```

Comment:
subscriptionInfoWidget. Add Widget to end of all params, or please remove it all.